src/StudyRegistration/StudyRegistration.tsx

```diff
@@ -0,0 +1,376 @@
+      {
+        repository: formValues.repository || '',
+        repository_study_ids: ((!formValues.repository_study_ids || formValues.repository_study_ids[0] === '') ? [] : formValues.repository_study_ids),
+        clinical_trials_id: formValues.clinical_trials_id || '',
+      }).then((preprocessedMetadata) => createCEDARInstance(cedarUserUUID, preprocessedMetadata)
+      .then(() => registerStudyInMDS(studyID, preprocessedMetadata).then(() => setFormSubmissionStatus({ status: 'success' })),
+        (err) => setFormSubmissionStatus({ status: 'error', text: err.message })),
+    (err) => setFormSubmissionStatus({ status: 'error', text: err.message }));
+  };
+
+  const onFinish = (values) => {
+    handleRegisterFormSubmission(values);
+  };
+
+  const onReset = () => {
+    form.resetFields();
+  };
+
+  if (formSubmissionStatus) {
+    return (
+      <div className='study-reg-container'>
+        <div className='study-reg-form-container'>
+          {(formSubmissionStatus.status === 'success') ? (
+            <Result
+              status={formSubmissionStatus.status}
+              title='Your study has been registered!'
+              subTitle='Please allow up to 24 hours until the platform is updated'
+              extra={[
+                <Button type='primary' key='register' onClick={() => { setFormSubmissionStatus(null); setRegRequestPending(false); }}>
+                  Register Another Study
+                </Button>,
+                <Link key='discovery' to={'/discovery'}>
+                  <Button>Go To Discovery Page</Button>
+                </Link>,
+                <Tooltip title='Check the newly created CEDAR metadata instance on CEDAR platform. It should be available under the "Shared with Me" tab'>
+                  <Button href='https://cedar.metadatacenter.org/' target='_blank' rel='noreferrer'>
+                    <Space>
+                      Go To CEDAR <FontAwesomeIcon icon={'external-link-alt'} />
+                    </Space>
+                  </Button>
+                </Tooltip>,
+              ]}
+            />
+          ) : (
+            <Result
+              status={formSubmissionStatus.status}
+              title='A problem has occurred during registration!'
+              subTitle={formSubmissionStatus.text}
+              extra={[
+                <Button type='primary' key='close' onClick={() => { setFormSubmissionStatus(null); setRegRequestPending(false); }}>
+                  Close
+                </Button>,
+              ]}
+            />
+          )}
+        </div>
+      </div>
+    );
+  }
+
+  return (
+    <div className='study-reg-container'>
+      <div className='study-reg-form-container'>
+        <Form className='study-reg-form' {...layout} form={form} name='study-reg-form' onFinish={onFinish} validateMessages={validateMessages}>
+          <Divider plain>Registration Information</Divider>
+          <Form.Item
```

Comment:
may want to add a note that * fields are required

Reply:
done